### PR TITLE
Inheart 151 - Add AuctionHearts molecule

### PR DESCRIPTION
**Pull request by BeataKula**
Added AuctionHearts component 
Added AuctionHearts to stories
Added base AuctionHearts test 

image

image

### CODE REVIEW

File: libs/ui/molecules/src/lib/AuctionHearts/__tests__/AuctionHearts.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import { NativeBaseProvider } from 'native-base';
+
+describe('AuctionHearts', () => {
+  it('should render successfully auction hearts', () => {
+    const { container } = render(
+      <NativeBaseProvider>
+        <AuctionHearts price='1000' />
+      </NativeBaseProvider>
+    );
+    expect(container).toBeTruthy();
```
Comment (zgr92) [inline libs/ui/molecules/src/lib/AuctionHearts/__tests__/AuctionHearts.spec.tsx:12]: Do we plan to add proper tests later? Can we do at least one here?

File: libs/ui/molecules/src/lib/AuctionHearts/__stories__/AuctionHearts.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Column, NativeBaseProvider } from 'native-base';
+import { storiesOf } from '@storybook/react-native';
+import { AuctionHearts } from '../AuctionHearts';
+
+storiesOf('AuctionHearts', module).add('Basic', () => (
+  <NativeBaseProvider>
```
Comment (konradhanus) [inline libs/ui/molecules/src/lib/AuctionHearts/__stories__/AuctionHearts.stories.tsx:7]: - [ ] - use decorator
`.addDecorator((story) => <NativeBaseProvider>{story()}</NativeBaseProvider>)`

File: libs/ui/molecules/src/lib/AuctionHearts/AuctionHearts.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    <Row space={2} alignItems='center'>
+      <Image source={vector} alt='auction hearts' />
+      <Text fontSize='xl' style={styles.textStyle}>
+        {price}
+      </Text>
+    </Row>
+  );
+}
+
+const styles = StyleSheet.create({
```
Comment (zgr92) [inline libs/ui/molecules/src/lib/AuctionHearts/AuctionHearts.tsx:25]: We have styled-components in our package.json. Don't we want to use them here?

**Review by zgr92 — APPROVED** [review]

File: libs/ui/molecules/src/lib/AuctionHearts/AuctionHearts.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
+/* eslint-disable @nrwl/nx/enforce-module-boundaries */
+import React from 'react';
+import { Image } from '@inheartive/ui/atoms';
+import { vector } from '@inheartive/assets';
+import { Text, Row } from '@inheartive/ui/atoms';
+import { StyleSheet } from 'react-native';
+
+interface IAuctionHeartsProps {
+  price: number;
```
Comment (pmusialik-intive) [inline libs/ui/molecules/src/lib/AuctionHearts/AuctionHearts.tsx:9]: Is it price of AuctionHearts or their quantity? :)

File: libs/ui/molecules/src/lib/AuctionHearts/__stories__/AuctionHearts.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+    <Column space={4} mt={4} mx='auto'>
+      <AuctionHearts price='10 000' />
+      <AuctionHearts price='5454' />
+      <AuctionHearts price='1000' />
+      <AuctionHearts price='0' />
```
Comment (pmusialik-intive) [inline libs/ui/molecules/src/lib/AuctionHearts/__stories__/AuctionHearts.stories.tsx:12]: Should be number passed, but there is a string value instead, right?

File: libs/ui/molecules/src/lib/AuctionHearts/__tests__/AuctionHearts.spec.tsx [hunk 6]
```diff
@@ -0,0 +1,14 @@
+import { render } from '@testing-library/react-native';
+import { AuctionHearts } from '../AuctionHearts';
+import { NativeBaseProvider } from 'native-base';
+
+describe('AuctionHearts', () => {
+  it('should render successfully auction hearts', () => {
+    const { container } = render(
+      <NativeBaseProvider>
+        <AuctionHearts price='1000' />
```
Comment (pmusialik-intive) [inline libs/ui/molecules/src/lib/AuctionHearts/__tests__/AuctionHearts.spec.tsx:9]: Should be number passed, but there is a string value instead, right?

File: apps/mobile/src/app/assets/index.ts [hunk 7]
```diff
@@ -1,4 +1,5 @@
 import logo from './images/logo.png';
 import textLogo from './images/text-logo.png';
+import vector from './images/vector.png';
```
Comment (pmusialik-intive) [inline apps/mobile/src/app/assets/index.ts:3]: Could we have some better name for this image?

**Pull request merged**